pkg/antctl/transform/networkpolicy/transform.go

```diff
@@ -20,34 +20,70 @@ import (
 	"sort"
 	"strconv"
 
+	"k8s.io/apimachinery/pkg/api/meta"
+	"k8s.io/apimachinery/pkg/runtime"
+	"k8s.io/kubectl/pkg/cmd/get"
+	"k8s.io/kubectl/pkg/scheme"
+	"k8s.io/utils/strings/slices"
+
 	"antrea.io/antrea/pkg/antctl/transform"
 	"antrea.io/antrea/pkg/antctl/transform/common"
 	cpv1beta "antrea.io/antrea/pkg/apis/controlplane/v1beta2"
-	"antrea.io/antrea/pkg/controller/networkpolicy"
 )
 
 type Response struct {
 	*cpv1beta.NetworkPolicy
 }
 
+type NPSorter struct {
+	networkPolicies []cpv1beta.NetworkPolicy
+	sortBy          string
+}
+
 func objectTransform(o interface{}, _ map[string]string) (interface{}, error) {
 	return Response{o.(*cpv1beta.NetworkPolicy)}, nil
 }
 
 func listTransform(l interface{}, opts map[string]string) (interface{}, error) {
+	specialCases := []string{
+		"effectivePriority",
+	}
```

Comment:
I feel you can define a string constant `sortByEffectivePriority = effectivePriority` on the top of this file instead of a slice here.

Reply:
@antoninbas suggested to use slice, so that in future if any new cases are added we do not need to modify the code much and just we can add the sort case for the newly added case and it will work.

Reply:
OK, no strong opinion, please address other comments, I will mark this resolved.